libs/pmp-web/repository/data-access-single-repository-statistics/src/lib/pmp-web-repository-data-access-single-repository-statistics.module.ts

```diff
@@ -0,0 +1,26 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { StoreModule } from '@ngrx/store';
+import { EffectsModule } from '@ngrx/effects';
+import {
+  SINGLEREPOSITORYSTATISTICS_FEATURE_KEY,
+  initialState as singleRepositoryStatisticsInitialState,
+  singleRepositoryStatisticsReducer
+} from './+state/single-repository-statistics.reducer';
+import { SingleRepositoryStatisticsEffects } from './+state/single-repository-statistics.effects';
+import { SingleRepositoryStatisticsFacade } from './+state/single-repository-statistics.facade';
+import { SingleRepositoryStatisticsDataService } from './services/single-repository-statistics-data.service';
+
+@NgModule({
+  imports: [
+    CommonModule,
+    StoreModule.forFeature(
+      SINGLEREPOSITORYSTATISTICS_FEATURE_KEY,
+      singleRepositoryStatisticsReducer,
+      { initialState: singleRepositoryStatisticsInitialState }
+    ),
+    EffectsModule.forFeature([SingleRepositoryStatisticsEffects])
+  ],
+  providers: [SingleRepositoryStatisticsFacade, SingleRepositoryStatisticsDataService]
+})
+export class PmpWebRepositoryDataAccessSingleRepositoryStatisticsModule {}
```

Comment:
Was that a good idea to split data access to a single repository and to a collection of statistics?

Reply:
Currently I don't see any drawbacks. I don't need code from collection and that @Stefaneek will refactor it to grouped and clean structure